datasets/lccc/README.md

```diff
@@ -0,0 +1,156 @@
+  - [Considerations for Using the Data](#considerations-for-using-the-data)
+    - [Social Impact of Dataset](#social-impact-of-dataset)
+    - [Discussion of Biases](#discussion-of-biases)
+    - [Other Known Limitations](#other-known-limitations)
+  - [Additional Information](#additional-information)
+    - [Dataset Curators](#dataset-curators)
+    - [Licensing Information](#licensing-information)
+    - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://github.com/thu-coai/CDial-GPT
+- **Repository:** https://github.com/thu-coai/CDial-GPT
+- **Paper:** https://arxiv.org/abs/2008.03946
+
+### Dataset Summary
+
+lccc: Large-scale Cleaned Chinese Conversation corpus (LCCC) is a large Chinese dialogue corpus originate from Chinese social medias. A rigorous data cleaning pipeline is designed to ensure the quality of the corpus. This pipeline involves a set of rules and several classifier-based filters. Noises such as offensive or sensitive words, special symbols, emojis, grammatically incorrect sentences, and incoherent conversations are filtered.
+
+lccc是一套来自于中文社交媒体的对话数据，我们设计了一套严格的数据过滤流程来确保该数据集中对话数据的质量。 这一数据过滤流程中包括一系列手工规则以及若干基于机器学习算法所构建的分类器。 我们所过滤掉的噪声包括：脏字脏词、特殊字符、颜表情、语法不通的语句、上下文不相关的对话等。
+
+### Supported Tasks and Leaderboards
+
+- dialogue-generation: The dataset can be used to train a model for generating dialogue responses.
+- response-retrieval: The dataset can be used to train a reranker model that can be used to implement a retrieval-based dialogue model.
+
+### Languages
+
+LCCC is in Chinese
+
+LCCC中的对话是中文的
+
+## Dataset Structure
+
+### Data Instances
+
+["火锅 我 在 重庆 成都 吃 了 七八 顿 火锅", "哈哈哈哈 ！ 那 我 的 嘴巴 可能 要 烂掉 ！", "不会 的 就是 好 油腻"]
+
+### Data Fields
+
+Each line is a list of utterances that consist a dialogue.
+Note that the LCCC dataset provided in our original Github page is in json format,
+however, we are providing LCCC in jsonl format here.
+
+### Data Splits
+
+We do not provide the offical split for LCCC-large.
+But we provide a split for LCCC-base:
+
+|train|valid|test|
+|:---:|:---:|:---:|
+|6,820,506 | 20,000 | 10,000|
+
+## Dataset Creation
+
+### Curation Rationale
+
+[Needs More Information]
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+[Needs More Information]
+
+#### Who are the source language producers?
+
+[Needs More Information]
+
+### Annotations
+
+#### Annotation process
+
+[Needs More Information]
+
+#### Who are the annotators?
+
+[Needs More Information]
+
+### Personal and Sensitive Information
+
+[Needs More Information]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[Needs More Information]
+
+### Discussion of Biases
+
+[Needs More Information]
+
+### Other Known Limitations
+
+[Needs More Information]
+
+## Additional Information
+
+### Dataset Curators
+
+[Needs More Information]
+
+### Licensing Information
+
+[Needs More Information]
+
+### Citation Information
+
+[Needs More Information]
```

Comment:
Missing information: please add the citation between triple backticks
```suggestion
@inproceedings{wang2020chinese,
  title={A Large-Scale Chinese Short-Text Conversation Dataset},
  author={Wang, Yida and Ke, Pei and Zheng, Yinhe and Huang, Kaili and Jiang, Yong and Zhu, Xiaoyan and Huang, Minlie},
  booktitle={NLPCC},
  year={2020},
  url={https://arxiv.org/abs/2008.03946}
}

### Contributions

Thanks to [@github-username](https://github.com/<github-username>) for adding this dataset.
```
Also please fill the Contributions section with your GitHub username.